logging/monolog_email.rst

```diff
@@ -7,7 +7,7 @@ How to Configure Monolog to Email Errors
 .. caution::
 
     This feature is not compatible yet with the new :doc:`Symfony mailer </mailer>`,
-    so it requires using SwiftMailer.
+    so it requires using :doc:`SwiftMailer </email>`.
```

Comment:
We at Symfony want the new Mailer is used, so adding a link and make it more simple to go to SwiftMailer does not makes sense to me.

Reply:
@OskarStark actually, monolog isn't compatible with Mailer. That's why i add the link 😉

Reply:
the PR has been merged recently
https://github.com/symfony/symfony/pull/33456
so IMO ne more need to promote swiftmailer email logging

exept for older version maybe :)